casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala

```diff
@@ -60,4 +76,40 @@ class RholangBuildTest extends FlatSpec with Matchers {
       }
   }
 
+  "Our build system" should "execute the genesis block" ignore effectTest {
+    val REV_ADDRESS_COUNT = 16000
+
+    val validatorKeyPairs = (1 to 4).map(_ => Secp256k1.newKeyPair)
+    val bonds = validatorKeyPairs
+      .map(_._2)
+      .zipWithIndex
+      .map { case (v, i) => v -> (2L * i.toLong + 1L) }
+      .toMap
+    val vaults = (1 to REV_ADDRESS_COUNT)
+      .map(i => (Secp256k1.newKeyPair, i))
+      .map { case ((_, publicKey), i) => Vault(RevAddress.fromPublicKey(publicKey).get, i.toLong) }
+      .toSeq
+    val genesisParams: GenesisParameters = (
+      validatorKeyPairs,
+      Genesis(
+        shardId = "MultiParentCasperSpec",
+        timestamp = 0L,
+        proofOfStake = ProofOfStake(
+          minimumBond = 0L,
+          maximumBond = Long.MaxValue,
+          epochLength = 1,
+          validators = bonds.map(Validator.tupled).toSeq
+        ),
+        genesisPk = defaultPub,
+        vaults = vaults,
+        supply = Long.MaxValue
+      )
+    )
```

Comment:
there's no need to duplicate all the default values. You can simply:
```
val genesisParams = defaultGenesisParams().copy(vaults = vaults)
```

Reply:
PTAL